lib/game_controller.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'board'
+
+class GameController
+  def start_game
+    board = Board.new
+    puts "Welcome to Connect 4!!!"
+    puts "Enter P to start a game, or Q to quit."
+    user_input = gets.chomp
```

Comment:
can maybe throw downcase back in here to account for any inputs such as P or Q

Reply:
oops, we forgot to put that back in after debugging. fixed 👍